packages/java/endpoint/src/main/java/dev/hilla/springnative/HillaHintsRegistrar.java

```diff
@@ -19,8 +30,13 @@
  */
 public class HillaHintsRegistrar implements RuntimeHintsRegistrar {
 
+    private static final String GENERATED_OPENAPI_JSON = "target/generated-resources/openapi.json";
```

Comment:
Since we’re in runtime, I suppose you could get this file from resources. Would this work?

Reply:
No, and if you compile a project then `openapi.json` is not available in the jar either